axelrod/tests/unit/test_tournament.py

```diff
@@ -89,7 +89,7 @@ def test_warning(self):
             repetitions=1)
         with warnings.catch_warnings(record=True) as w:
             # Check that a warning is raised if no results set is built and no
```

Comment:
"Check that a warning is raised if **no results set is built** and **no** filename **not** given".